include/aspect/postprocess/interface.h

```diff
@@ -226,10 +230,34 @@ namespace aspect
          * if one of them has the desired type specified by the template
          * argument. If so, return a pointer to it. If no postprocessor is
          * active that matches the given type, return a NULL pointer.
+         *
+         * @deprecated Use has_matching_postprocessor() and
+         * get_matching_postprocessor() instead.
          */
         template <typename PostprocessorType>
         PostprocessorType *
-        find_postprocessor () const;
+        find_postprocessor () const DEAL_II_DEPRECATED;
+
+        /**
+         * Go through the list of all postprocessors that have been selected
+         * in the input file (and are consequently currently active) and return
+         * true if one of them has the desired type specified by the template
+         * argument.
+         */
+        template <typename PostprocessorType>
+        bool
+        has_matching_postprocessor () const;
+
+        /**
+         * Go through the list of all postprocessors that have been selected
+         * in the input file (and are consequently currently active) and see
+         * if one of them has the desired type specified by the template
```

Comment:
" or can be casted to that type"? This might be important if we have additional base classes.